schema.json

```diff
@@ -113,20 +113,86 @@
           "description": "Name of the referenced table",
           "type": "string"
         },
+        "match_type": {
+          "description": "Match type of the foreign key constraint",
+          "$ref": "#/$defs/ForeignKeyMatchType",
+          "default": "SIMPLE"
+        },
         "on_delete": {
           "description": "On delete behavior of the foreign key constraint",
-          "$ref": "#/$defs/ForeignKeyReferenceOnDelete",
+          "$ref": "#/$defs/ForeignKeyOnDelete",
+          "default": "NO ACTION"
+        },
+        "on_update": {
+          "description": "On update behavior of the foreign key constraint",
+          "$ref": "#/$defs/ForeignKeyOnDelete",
```

Comment:
Looks like the wrong `$ref` here.

Reply:
No, it is on purpose. `ON UPDATE` and `ON DELETE` have the same actions, so we can reuse it in both places. But I can rename it to make clearer that it is intentional.